shell/browser/native_window.cc

```diff
@@ -54,7 +84,19 @@ NativeWindow::NativeWindow(const gin_helper::Dictionary& options,
   options.Get(options::kFrame, &has_frame_);
   options.Get(options::kTransparent, &transparent_);
   options.Get(options::kEnableLargerThanScreen, &enable_larger_than_screen_);
-  options.Get(options::ktitleBarOverlay, &titlebar_overlay_);
+
+  v8::Local<v8::Value> titlebar_overlay;
+  options.Get(options::ktitleBarOverlay, &titlebar_overlay);
+  options.Get(options::kTitleBarStyle, &title_bar_style_);
+
+  if (!titlebar_overlay.IsEmpty() && titlebar_overlay->IsBoolean()) {
```

Comment:
Instead of checking `IsEmpty`, we should put this stuff behind the `Get` call.  It returns a success boolean. E.g.

```
if (options.Get(thing, &overlay)) {
  // Do something with overlay, it is safe here
}